.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: python
+python:
+    - "2.7"
```

Comment:
Worked around by checking the Python version. I don't feel great about branching on the language version, but the error message is helpful in either case ("too few arguments" in 2.7 versus "the following arguments are required: group_name" in 3.5) and it seems useful to assert that the error messages are correct.